helpdesk_mgmt/migrations/13.0.1.7.0/post-migrate.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.delete_record_translations(
+        env.cr, "helpdesk_mgmt", ["closed_ticket_template", "changed_stage_template"]
```

Comment:
This should go out, as it can remove legit translations (as this is executed after good translations loading). Do it on pre or not do it at all.